Change get_simsteps function to handle NVT equilibration in plain MD protocol

hannahbaumann opened the pull request:
## Developers certificate of origin
- [x] I certify that this contribution is covered by the MIT License [here](https://github.com/OpenFreeEnergy/openfe/blob/main/LICENSE) and the **Developer Certificate of Origin** at <https://developercertificate.org/>.

pep8speaks commented:
Hello @hannahbaumann! Thanks for opening this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:





* In the file [`openfe/protocols/openmm_utils/settings_validation.py`](https://github.com/OpenFreeEnergy/openfe/blob/bccfb233c6403de00cc681bc2713d6c84b5a18d1/openfe/protocols/openmm_utils/settings_validation.py):

> [Line 69:17](https://github.com/OpenFreeEnergy/openfe/blob/bccfb233c6403de00cc681bc2713d6c84b5a18d1/openfe/protocols/openmm_utils/settings_validation.py#L69): [E222](https://duckduckgo.com/?q=pep8%20E222) multiple spaces after operator

IAlibay commented:
@hannahbaumann could you tick that box re: certifying contributions?

hannahbaumann commented:
@IAlibay : Addressed your comments! I realized that a lot of tests are failing in the plain MD PR because I had already changed the `get_sim_steps` function for the plain MD protocol, so the RFE and HFE tests are failing, probably best to fix/merge after this merge?!

IAlibay reviewed (approved):
lgtm!
